openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -349,6 +351,55 @@ def test_asymmetric_jwt_access_token(self):
         """
         self._test_jwt_access_token('dot_app', token_type='jwt', grant_type='password', asymmetric_jwt=True)
 
+    @override_waffle_switch(DISABLE_JWT_FOR_MOBILE, False)
+    @patch('openedx.core.djangoapps.oauth_dispatch.views.is_request_from_mobile_app')
+    def test_disable_jwt_waffle_switch_disabled_non_mobile(self, mock_is_request_from_mobile_app):
+        """
+        DISABLE_JWT_FOR_MOBILE disabled
+        Non-mobile request
+        Test JWT token is returned
+        """
+        mock_is_request_from_mobile_app.return_value = False
+        self._test_jwt_access_token('dot_app', token_type='jwt', grant_type='password')
```

Comment:
we should set asymmetric jwt True for mobile.

Reply:
Added